grassroots-shared/src/dtos/PhoneCanvass/CallStatus.dto.ts

```diff
@@ -20,3 +25,20 @@ export function CallStatusDecorator(): PropertyDecorator {
     ApiProperty({ enum: CallStatusEnum }),
   );
 }
+
+enum CallResultEnum {
+  CANCELED = "CANCELED",
+  COMPLETED = "COMPLETED",
+  BUSY = "BUSY",
+  NO_ANSWER = "NO_ANSWER",
+  FAILED = "FAILED",
+}
+
+export type CallResult = keyof typeof CallResultEnum;
+
+export function CallResultDecorator(): PropertyDecorator {
```

Comment:
I'm not seeing where this is used. Was the idea to use this in the Entity?

Reply:
This will be required to pass a CallResult to the frontend. Maybe we won't need this though?

Will call party participants want to know if a call was removed from the queue because it was busy etc?

That does sound probably post MVP, in which case I can remove this.